docs/articles/documentation/guides/advanced-guides/authentication.md

```diff
@@ -134,50 +111,38 @@ test('Anonymous users can see newly created comments', async t => {
         .click('#submit')
         .useRole(Role.anonymous());
 
-        const comment = await Selector('#comment-data');
+    const comment = await Selector('#comment-data');
 
-        await t.expect(comment.innerText).eql('Hey ya!');
+    await t.expect(comment.innerText).eql('Hey ya!');
 });
 ```
 
-## Role Options
-
-You can pass the following options to the [Role constructor](#create-and-apply-roles).
-
-### options.preserveUrl
-
-Use this option to control which page is opened after you switch to the role.
-
-By default, TestCafe navigates back to the page that was opened previously to switching to the role.
-Set the `preserveUrl` option to `true` to save the URL to which the browser was redirected after logging in.
-TestCafe will navigate to the saved URL each time after you switch to this role.
```

Comment:
Change to:
After you switch to this role, TestCafe will navigate to the saved URL each time.